javascript/ql/src/semmle/javascript/StandardLibrary.qll

```diff
@@ -154,7 +154,7 @@ private class ES2015PromiseDefinition extends PromiseDefinition, DataFlow::NewNo
 /**
  * A promise that is resolved with the given value.
  */
-abstract class ResolvedPromiseDefinition extends DataFlow::CallNode {
+abstract class PromiseCreationCall extends DataFlow::CallNode {
```

Comment:
This is a user-visible API change. You could avoid this by making `PremiseCreationCall` a superclass of `ResalvedPromiseDefinition` instead.

Reply:
Otherwise we have to keep the old name around as a deprecated alias for at least one release cycle.